testworld/park.go

```diff
@@ -195,16 +195,17 @@ func (r *hostManager) startTempHost(name string) error {
 	return nil
 }
 
-func (r *hostManager) createHost(name, twConfigName, p2pTimeout string, apiPort, p2pPort int64, createConfig, multiAccount bool, bootstraps []string) *host {
+func (r *hostManager) createHost(name, twConfigName, p2pTimeout string, apiPort, p2pPort, grpcPort int64, createConfig, multiAccount bool, bootstraps []string) *host {
 	return newHost(
 		name,
 		r.ethNodeUrl,
 		r.accountKeyPath,
 		r.accountPassword,
 		r.network,
+		"127.0.0.1",
 		twConfigName,
 		p2pTimeout,
-		apiPort, p2pPort, bootstraps,
+		apiPort, p2pPort, grpcPort, bootstraps,
```

Comment:
I usually don't care much about whats here but let's not have this on testworld hosts, it just creates unnecessary dependancies. Specially since we will remove grpc soon.

Reply:
I dont understand what you mean by lets not have it as part of testworld. We are seperating the HTTP api from GRPC. So wee need to have a seperate port for GRPC. I'm open any solutions you have